Shared/Mirror/mirrorReflection.js

```diff
@@ -0,0 +1,36 @@
+//This script acts on the reflection box in front of the mirror so that when and avatar
```

Comment:
Missing file headers with copyright notification, creator and optional short description of the script, see other scripts in repo for examples.